src/operator/tensor/matrix_op.cc

```diff
@@ -222,7 +222,7 @@ NNVM_REGISTER_OP(slice)
 
 .. note:: ``crop`` is deprecated. Use ``slice`` instead.
 
-This function returns a sliced continous region of the array between the indices given 
+This function returns a sliced continous region of the array between the indices given
```

Comment:
not sure if it is correct. `slice` is edited by @nswamy. I just fixed a indentation error so that it can render correctly on the website.